Entities/Vehicles/Common/WoodVehicleDamages.as

```diff
@@ -50,7 +50,7 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 		{
 			if (damageowner.getTeamNum() != this.getTeamNum())
 			{
-				SendGameplayEvent(createVehicleDamageEvent(damageowner, false));
+				SendGameplayEvent(createVehicleDamageEvent(damageowner, damage));
```

Comment:
looks like this should use `dmg` instead of `damage` so you're basing the coins off the actual damage being done, and you don't get 4x as much money for hitting it with a sword

```suggestion
				SendGameplayEvent(createVehicleDamageEvent(damageowner, dmg));
```